website/docs/reference/transforms/lua.md.erb

```diff
@@ -40,116 +40,255 @@
 
 <TabItem value="add_fields">
```

Comment:
I like the changes in this file. I'm going to add a new "Guides" section that links to associated guides. This will help users discover your new aggregation guide.